### PR TITLE
Spring Boot template for service

### PR DESCRIPTION
**Pull request by duraz0rz**

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (phasebash) [inline .idea/compiler.xml]: We should probably not include `.idea` files

Comment (d--) [reply .idea/compiler.xml]: 👍

File: src/test/java/org/doge/DogeTasksServiceApplicationTests.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class DogeTasksServiceApplicationTests {
+
+	@Test
```
Comment (phasebash) [inline src/test/java/org/doge/DogeTasksServiceApplicationTests.java]: Probably want to test that you get a webapplicationcontext

File: .idea/compiler.xml (hunk 1 above)
Comment (duraz0rz) [reply .idea/compiler.xml]: Just went with what I found here: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems

Unless we want to be IDE-agnostic for this, then that makes sense

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+sourceCompatibility = 1.8
+
+repositories {
+	mavenCentral()
+}
+
+
+dependencies {
+	compile('org.springframework.boot:spring-boot-starter-jersey')
+	compileOnly('org.projectlombok:lombok')
```
Comment (phasebash) [inline build.gradle:31]: where does this configuration come from?

**Review by phasebash — APPROVED** [review]

File: build.gradle (hunk 3 above)
Comment (duraz0rz) [reply build.gradle:31]: I selected it from http://start.spring.io

**Pull request merged**